.gitlab/ci/on-push.yml

```diff
@@ -78,16 +78,6 @@ create-instances:
         ./Tests/scripts/prepare_content_packs_for_testing.sh "$GCS_MARKET_BUCKET"
       fi
     - section_end "Prepare Content Packs for Testing"
-    - section_start "Zip Marketplace Packs"
-    - |
-      if [[ $CI_COMMIT_BRANCH != master ]] && [[ $CI_COMMIT_BRANCH != 21\.* ]] && [[ $CI_COMMIT_BRANCH != 22\.* ]]; then
-        echo "Skipping packs download to artifact on branch that is not master or a release branch"
-        echo "CI_COMMIT_BRANCH=$CI_COMMIT_BRANCH"
-      else
-        ZIP_FOLDER=$(mktemp -d)
-        python3 ./Tests/Marketplace/zip_packs.py -b 'marketplace-dist' -z $ZIP_FOLDER -a $ARTIFACTS_FOLDER -s $GCS_MARKET_KEY
-      fi
-    - section_end "Zip Marketplace Packs"
```

Comment:
Can leave this here and just use the condition of `if [[ -n "${BUCKET_UPLOAD}" ]]; then` for when to this should run since the `create-instances` job in the `bucket-upload` flow inherits from this job. Therefore there is no need to add this step in the `upload-packs-to-marketplace` job.

Reply:
So I should remove the entire _Zip Content Packs From GCS_ from `bucket-upload.yml` ?